ingest/lambdas/rss-feed-ingester.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable import/no-unresolved, import/extensions */
+// @ts-check
+const pMap = require('p-map');
+
+const dspaceClient = require('../../lib/dspace-client');
+const ingestQueue = require('../lib/ingest-queue');
+const { getStringFromEnv } = require('../../lib/env-utils');
+
+const dspaceFeedReadInterval = process.env.DSPACE_FEED_READ_INTERVAL
+  ? Number.parseInt(process.env.DSPACE_FEED_READ_INTERVAL)
+  : 300;
+
+/**
+ * Returns a boolean indiciting whether or not we shoudl publish documents
+ * from the RSS feed based on when the RSS feed was published and the
+ * last time we checked it. Set the `feedReadInterval` to -1 to always
+ * pass this test.
+ *
+ * @param {Date} publishedDate The published date of the RSS feed.
+ * @param {number} feedReadInterval The number in seconds since the
+ * published RSS feed was last read.
+ *
+ * @returns {boolean} Whether or not we should publish new documents based
+ *          on the last time we checked the RSS feed
+ */
+const shouldQueuePublishedDocuments = (publishedDate, feedReadInterval) => {
```

Comment:
Changed it to `Date.now() - feedReadInterval` and added a comment. I'll file an Issue in Github though because we shouldn't be relying on the read interval but instead save off the "last read date" somewhere and use that for this check.